python/Ganga/new_tests/GPI/newXMLTest/TestXMLGenAndLoad.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import absolute_import
+
+from ..GangaUnitTest import GangaUnitTest
+
+from os import path, stat
+
+import time
+
+def getJobsPath():
```

Comment:
This function is repeated from the other file. Perhaps move them to a common place or into _init__.py?